tools/anonymizer_istex.js

```diff
@@ -0,0 +1,41 @@
+/*
+*  Tool to anonymize logs file
+*  It reads line by line the file and remove unwanted values
+*
+*/
+
+
+var fs = require('fs');
+var InputLogName;
+var OutputLogName;
```

Comment:
Rule of thumb : no uppercased letter at the beginning of variable names, unless it refers to a constructor function.